core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/InvokerReactive.scala

```diff
@@ -299,4 +300,8 @@ class InvokerReactive(
     }
   })
 
+  override def enable(): Route = ???
```

Comment:
I haven't tried it but would it be ok to leave these methods empty to run it?

Reply:
Sorry, you mean implement it like below?
```
override def enable(): Route = {
    complete("todo or pong")
}
```
Due to InvokerReactive extends InvokerCore which has `enabled/disabled`,
so need to implement enable/disabled

Reply:
when deploy using `InvokerReactive` and send `enable/disable` requests to invoker, what will happen?